doc/telescope.txt

```diff
@@ -269,20 +269,530 @@ actions.open_qflist()                                  *actions.open_qflist()*
 ================================================================================
                                                              *telescope.builtin*
 
-A collection of builtin pickers for telescope.
-
-Meant for both example and for easy startup.
+Telescope Builtins is a collection of community maintained pickers to support
+common workflows. It can be used as reference when writing PRs, Telescope
+extensions, your own custom pickers, or just as a discovery tool for all of the
+amazing pickers already shipped with Telescope!
 
 Any of these functions can just be called directly by doing:
 
-:lua require('telescope.builtin').$NAME()
+:lua require('telescope.builtin').$NAME_OF_PICKER()
+
+To use any of Telescope's default options or any picker-specific options, call
+your desired picker by passing a lua table to the picker with all of the
+options you want to use. Here's an example with the live_grep picker:
+
+:lua require('telescope.builtin').live_grep({
```

Comment:
Maybe we can get here another example here on how it would work with dropdown/themes and additional opts.

See last discussion in gitter: https://gitter.im/nvim-telescope/community

```lua
require'telescope.builtin'.find_files(require('telescope.themes').get_dropdown({
  key = value, 
  ...
})
```

Reply:
Sure! I can again have that included in the next PR with updates and cleanup to the README.

Reply:
![image](https://user-images.githubusercontent.com/37027883/118101240-e6158a80-b3c6-11eb-825a-5c8d5a3d073b.png)

I'm just adding this here because I apparently can't attach screenshots to Github project kanban boards, and want a reference to what we were talking about here that I can link to there to track this